### ISSUE TITLE
Post Thumbnail not returned on Parent Entity

### ISSUE THREAD

**Issue by garypaul** [pods-framework/pods#3779]
When you create a 1:M relationship and are in the child entity, you can't seem to access the parent's post_thumbnail property.

> Example.
> Two pods. Department ( parent ) and test_person ( children )
> Create the `department` with a featured Image and a relationship with `test_person`

<img width="1238" alt="1_department_pod_edit" src="https://cloud.githubusercontent.com/assets/1945849/18364739/4f2f730a-75c4-11e6-88b1-5bd90fdc5e6e.png">
<img width="975" alt="2_department_pod_saved" src="https://cloud.githubusercontent.com/assets/1945849/18364747/4f65e962-75c4-11e6-9669-adc1298a21ca.png">
<img width="736" alt="3_test_department_editing" src="https://cloud.githubusercontent.com/assets/1945849/18364743/4f55cb7c-75c4-11e6-9146-80a37f6f6b2d.png">
## Create the `test_person` with a link to `department`

<img width="750" alt="4_test_person_pod" src="https://cloud.githubusercontent.com/assets/1945849/18364742/4f53ce58-75c4-11e6-94be-777a2eb576a6.png">
<img width="566" alt="5_test_person_editing" src="https://cloud.githubusercontent.com/assets/1945849/18364745/4f5c0e6a-75c4-11e6-9bd7-6d572137e062.png">
## Create the template to display a single `test_person`

<img width="1070" alt="6_test_template_2_autocomplete" src="https://cloud.githubusercontent.com/assets/1945849/18364746/4f615514-75c4-11e6-98f2-a133de6765a0.png">
<img width="600" alt="7_test_template_1" src="https://cloud.githubusercontent.com/assets/1945849/18364741/4f4442da-75c4-11e6-9376-2e9c382fdc09.png">
## The **featured Image** of the `department` doesn't show up. (ToT) 👎

<img width="462" alt="8_template_output" src="https://cloud.githubusercontent.com/assets/1945849/18364744/4f59b8fe-75c4-11e6-8ef9-0cda11bba95c.png">

If you noticed, it doesn't show up in the **reference** or **autocomplete** either, which is potentially where the issue is.

Temporary fix would be... don't use Featured Image and add an Image field... though `post_thumbnail` is such an integral part of the whole WP ecosystem that it seems silly to not support that. 

Is it possible that this was done intentionally to avoid an extra database table join? 

Here's the template code I used:

```
<h1>{@post_title}</h1>

<p>Department Image Path: 
    {@department.post_thumbnail}
    {@department.post_thumbnail_url}
    {@department.post_thumbnail.thumbnail}
    {@department.post_thumbnail.medium}
</p>

{@post_content}
```

**Comment by quasel** [comment]
you could try 

```
[each department]
	{@post_thumbnail.url}
[/each]
```
which works for me all to well i got the image 24 times :P i rarely use the templates maybe i have time later on to dig deeper or somebody else jumps in ^^

**Comment by quasel** [comment]
That's still an issue - traversal is broken for any related pod like
` related_pod.post_thumbnail` or
` related_pod.post_thumbnail.ID`

**Comment by jimtrue** [comment]
Related Pods (multiple select) work in an `[each related]{@post_thumbnail}[/each]` block, so this is strictly limited to the traversal of post_thumbnail helper in traversals.

**Comment by quasel** [comment]
Workaround from jim - use multiple select and limit to 1

**Comment by jimtrue** [comment]
I can confirm this is an issue, even with Pods 2.7 RC1. The Template reference doesn't appear to extrapolate _anything_ from the single select relationship, but `single_relationship.post_title` outputs properly but the `single_relationship.post_thumbnail` doesn't.

**Comment by quasel** [comment]
ran into this again :/  it surfaced twice for the BB Integration! 

book_author relationship field  to another CPT 1:m -> 3 related items bidirectional assigned! 

portrait is a single select image! would expect both to work the same - weird it returns always a string for the first image and nothing for the rest 

<img width="866" alt="bildschirmfoto 2018-07-16 um 18 39 09 pm" src="https://user-images.githubusercontent.com/9464217/42771466-9d4c3ffa-8927-11e8-9ead-b7720d2a119a.png">

**Comment by JoryHogeveen** [comment]
@quasel 
To reply on your last comment.
Traversal over multiple selects simple isn't (and shouldn't be) possible like that.
At least not for magic tags...

The actual issue of this topic is fixed in #5610
